chapter_09_docker.asciidoc

```diff
@@ -72,6 +72,7 @@ I hope you enjoy the new version!
 ****
 
 === TDD and the Danger Areas of Deployment
+// XUAN: I read the title as meaning _TDD and_ the danger areas of deployment, but the following section seems to be talking about the danger areas of deployment?
```

Comment:
thanks @mmxw !  yes i think this dates back to a previous edition where i linked more explicitly between the danger areas and how tdd can help.  now i think we're more talking about how docker can help, by letting us take small steps and test things locally... with tdd still, but kind of later in the loop.

will update the title to remove the tdd bit probably.